lib/main.dart

```diff
@@ -1,115 +1,37 @@
+import 'package:deep_conference/application/logic/navigation_cubit.dart';
+import 'package:deep_conference/constants/theme_data.dart';
+import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter/material.dart';
-
-void main() {
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'application/logic/schedule_cubit.dart';
+import 'application/presentation/root_screen.dart';
+import 'data/firebase_options.dart';
+
+//na ovaj način se aplikacija povezuje na firebase
+Future main() async {
```

Comment:
Neka u `main.dart`-u ostane samo `main` metoda. `MyApp` widget premjestite u zaseban fajl pod nazivmo `app.dart`. I još nešto nemojte stavljati samo `Future` kao return type metoda, ovdje može biti `dynamic`, a inače ako nešto returna `Future`, onda napišite `Future<T>` Gdje "T" predstavlja neki data type jel :).